2024-02-05 22:08:04,534 INFO    MainThread:28787 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-02-05 22:08:04,534 INFO    MainThread:28787 [wandb_setup.py:_flush():76] Configure stats pid to 28787
2024-02-05 22:08:04,534 INFO    MainThread:28787 [wandb_setup.py:_flush():76] Loading settings from /Users/lihongxuan/.config/wandb/settings
2024-02-05 22:08:04,534 INFO    MainThread:28787 [wandb_setup.py:_flush():76] Loading settings from /Users/lihongxuan/Desktop/AIPI/Courses/AIPI540/AIPI-540/wandb/settings
2024-02-05 22:08:04,534 INFO    MainThread:28787 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-05 22:08:04,534 INFO    MainThread:28787 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-05 22:08:04,534 INFO    MainThread:28787 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-05 22:08:04,535 INFO    MainThread:28787 [wandb_init.py:_log_setup():526] Logging user logs to /Users/lihongxuan/Desktop/AIPI/Courses/AIPI540/AIPI-540/wandb/run-20240205_220804-7chzm0yg/logs/debug.log
2024-02-05 22:08:04,535 INFO    MainThread:28787 [wandb_init.py:_log_setup():527] Logging internal logs to /Users/lihongxuan/Desktop/AIPI/Courses/AIPI540/AIPI-540/wandb/run-20240205_220804-7chzm0yg/logs/debug-internal.log
2024-02-05 22:08:04,535 INFO    MainThread:28787 [wandb_init.py:init():566] calling init triggers
2024-02-05 22:08:04,535 INFO    MainThread:28787 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-02-05 22:08:04,535 INFO    MainThread:28787 [wandb_init.py:init():616] starting backend
2024-02-05 22:08:04,535 INFO    MainThread:28787 [wandb_init.py:init():620] setting up manager
2024-02-05 22:08:04,537 INFO    MainThread:28787 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-02-05 22:08:04,540 INFO    MainThread:28787 [wandb_init.py:init():628] backend started and connected
2024-02-05 22:08:04,570 INFO    MainThread:28787 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2024-02-05 22:08:04,570 INFO    MainThread:28787 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-02-05 22:08:04,570 INFO    MainThread:28787 [wandb_init.py:init():720] updated telemetry
2024-02-05 22:08:04,611 INFO    MainThread:28787 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-05 22:08:10,555 INFO    MainThread:28787 [wandb_run.py:_on_init():2254] communicating current version
2024-02-05 22:08:40,699 INFO    MainThread:28787 [wandb_init.py:init():804] starting run threads in backend
2024-02-05 22:08:52,949 INFO    MainThread:28787 [wandb_run.py:_console_start():2233] atexit reg
2024-02-05 22:08:52,949 INFO    MainThread:28787 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-02-05 22:08:52,949 INFO    MainThread:28787 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-02-05 22:08:52,950 INFO    MainThread:28787 [wandb_run.py:_redirect():2178] Redirects installed.
2024-02-05 22:08:52,950 INFO    MainThread:28787 [wandb_init.py:init():847] run started, returning control to user process
2024-02-05 22:45:11,035 INFO    MainThread:28787 [wandb_run.py:_finish():1962] finishing run aipi549/aipi540/7chzm0yg
2024-02-05 22:45:11,035 INFO    MainThread:28787 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-02-05 22:45:11,035 INFO    MainThread:28787 [wandb_run.py:_restore():2185] restore
2024-02-05 22:45:11,035 INFO    MainThread:28787 [wandb_run.py:_restore():2191] restore done
